pkgs/applications/networking/remote/freerdp/default.nix

```diff
@@ -58,7 +57,7 @@ stdenv.mkDerivation rec {
     ++ optional (cups != null)                "-DWITH_CUPS=ON"
     ++ optional (pcsclite != null)            "-DWITH_PCSC=ON"
     ++ optional buildServer                   "-DWITH_SERVER=ON"
-    ++ optional (optimize && stdenv.isx86_64) "-DWITH_SSE2=ON";
+    ++ optional (stdenv.isx86_64)             "-DWITH_SSE2=ON";
```

Comment:
Some cpus don’t support sse2 right?

Reply:
Oh nevermind all 64 bit cpus have it

Reply:
merged in https://github.com/NixOS/nixpkgs/pull/59230